src/lib/ext-activitylog.js

```diff
@@ -226,8 +245,14 @@ class View {
     this.logView.clearTable();
   }
 
-  renderHeading({ tabId }) {
-    this.logHeading.textContent = `Activity Logs Filtered By Tab Id: ${tabId}`;
+  updatePageType(heading) {
```

Comment:
This function is called only once, and its purpose in isolation is unclear. Let's remove the function and inline the logic at the caller.

Reply:
Removed the function and put the logic at the caller.